### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Taking an useful feature of github for PR templates.

See more here: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

### CODE REVIEW

**Comment by okurz** [comment]
I think it's a good idea in general. Maybe a bit too verbose. E.g. I would propose to just put one line `Related ticket: <url>`

**Comment by SergioAtSUSE** [comment]
Template simplified

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Description
```
Comment (okurz) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we need the section title when there is only one section?

**Comment by SergioAtSUSE** [comment]
@okurz, title removed

**Comment by okurz** [comment]
Looks good. Merged. We can always think about extending and improving.

**Pull request merged**